journal/updated/08-chapter-7.adoc

```diff
@@ -375,66 +376,66 @@ We were at meeting at Ballitore on first-day.
-by which our many bitters have been often sweetened,
+by which our many bitter waters have often been sweetened,
 and we have blessed that name which alone is worthy of thanksgiving and praise.
-I beg that we may ever rely upon, and live in pure obedience unto Him,
+I beg that we may ever rely upon and live in pure obedience unto Him
 who can subject our natural wills,
 and change the very nature of the lion into that of the lamb;
-and how pleasant is our dwelling, when upon God`'s holy mountain,
+and how pleasant is our dwelling when upon God`'s holy mountain,
 where nothing can hurt or destroy.
 
 --
 
 On third-day had a meeting at Baltiboys.
-
 We reached Dublin, and were at the Quarterly Meeting for the province,
-and at the National Half-year`'s Meeting;
-during the course of which we had some edifying seasons; but, alas! here,
-and almost everywhere else in this country, things are very low,
-and truth is trampled underfoot.
+and at the National Half-year`'s Meeting,
+during the course of which we had some edifying seasons. But, alas, here
+and almost everywhere else in this country things are very low,
+and truth is trampled underfoot!
 We were favoured with the company of William Rathbone, of Liverpool,
-and with that of many valuable Friends of this nation, of whom we took a long farewell.
+as well as that of many valuable Friends of this nation, of whom we took a long farewell.
 I had felt an engagement of mind to attend the Quarterly
-Meeting at Moyallen--the trial was great,
-as some Friends were going to England: my companion, too, was ill of a cold,
+Meeting at Moyallen; the trial was great,
+as some Friends were going to England. My companion, too, was ill of a cold,
 and not able to move away for a few days,
-so that we did not take all the meetings we had intended.
+so that we did not visit all the meetings we had intended to.
 I had, however, one more opportunity of sitting with Friends in Dublin,
 which was greatly to my satisfaction, and I took a solemn and final farewell of many.
 On the 15th of fifth month, we commenced our journey,
 having the company of our worthy friend, James Christy, who had waited for us.
-He has gone with us many long journeys:--I think I had his company six weeks,
+He has gone with us many long journeys. I think I had his company six weeks,
 and he is nearly eighty years of age.
 On sixth-day we reached his pleasant habitation.
-We then attended meetings at Lurgan, Grange, Ballyhagan and Moyallen; the Select Meeting,
+We then attended meetings at Lurgan, Grange, Ballyhagan and Moyallen, the Select Meeting
 on sixth-day, the Quarterly Meeting on seventh,
 and a very large meeting for worship on first-day,
 in all of which we were favoured with best help, and the conclusion was satisfactory.
 We took a solemn leave of many,
-and had some opportunities to edification and comfort with our beloved friends.
+and had some opportunities of edification and comfort with our beloved friends.
```

Comment:
her meaning here is more like: **and had some opportunities here with our beloved friends that were to our edification and comfort.**